src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{

```

Comment:
Please ensure proper spacing between the class name and the curly brace. It should be `Machine {`, not `Machine{`.